app/models/address.rb

```diff
@@ -11,7 +11,11 @@ def self.find_or_create_from_params(params)
   end
 
   validates :postcode, :address_line_1, presence: true
-  validates :point, presence: { message: 'geocoder lookup failed', if: ->(a) { a.postcode.present? } }
+  validates :point, presence: {
+    message: 'geocoder lookup from postcode failed',
```

Comment:
Could we use the `en.yml` for these error messages? A la https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/config/locales/en.yml#L40-L48